lcb_runner/prompts/code_generation.py

```diff
@@ -221,6 +222,32 @@ def format_prompt_generation(
         ]
         return chat_messages
 
+    if LanguageModelStyle == LMStyle.LLaMa2:
+        chat_messages = [
```

Comment:
There's a lot of shared code with llama2 and llama3 and mistral and yi, can we pull that into a helper function?